### DISCUSSION TITLE
react-pdf-viewer and pdfjs-dist unexpected token '||='

### DISCUSSION THREAD

**Discussion by ivanquirino** [vercel/next.js#35916] [Help]
My `next.config.js`:
```
require("dotenv").config();
const path = require('path');

module.exports = {
  reactStrictMode: true,
  images: {
    domains: ["firebasestorage.googleapis.com"]
  },
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    config.resolve.alias['pdfjs-dist'] = path.resolve( './node_modules/pdfjs-dist/legacy/build/pdf');
    return config;
  }
};
```
I'm getting this error:
![image](https://user-images.githubusercontent.com/3946025/161822467-e03569af-454a-4315-9bd1-70379a38a328.png)

Next version is 12.0.10

It should be resolving to the path specified in the config file. Has anyone worked with this setup, any solutions?

**Comment by kurves** [comment]
This error had me on a chokehold for days , but I resolved by it downgrading to pdfjs-dist@2.9.359

**Reply by ivanquirino** [reply]
Did you try all previous versions util 2.9.359 worked? That's a lot of work, thanks!

**Reply by icyJoseph** [reply]
Might be a good idea to mark this an answered and create an issue at https://github.com/mozilla/pdf.js
